x/ibc/04-channel/client/utils/utils.go

```diff
@@ -9,45 +9,57 @@ import (
 
 // QueryPacket returns a packet from the store
 func QueryPacket(
-	ctx client.CLIContext, portID, channelID string, sequence, timeout uint64, prove bool,
+	ctx client.CLIContext, portID, channelID string,
+	sequence, timeout uint64, queryRoute string,
```

Comment:
passing the query route is an anti-pattern

Reply:
How to change this function to not take the `queryRoute`?